DeepSpeech.py

```diff
@@ -1042,11 +1044,13 @@ def train():
     # Possibly restore checkpoint
     start_epoch = 0
     if restore_checkpoint:
+        # Restore checkpoint created after epoch N, restart from epoch N+1
         checkpoint = tf.train.get_checkpoint_state(checkpoint_dir)
         if checkpoint and checkpoint.model_checkpoint_path:
             hibernation_path = checkpoint.model_checkpoint_path
-            start_epoch = int(checkpoint.model_checkpoint_path.split('-')[-1])
-            print 'Resuming training from epoch %d' % (start_epoch + 1)
+            last_epoch = int(checkpoint.model_checkpoint_path.split('-')[-1])
+            start_epoch = last_epoch+1
```

Comment:
I've come up with a way to fix this, but I'm gonna wait for Tilman's distributed training changes to land first, since they'll help me quite a bit.